src/common/cluster-ipc.ts

```diff
@@ -1,59 +1,43 @@
-import { createIpcChannel } from "./ipc";
+import { handleRequest } from "./ipc";
 import { ClusterId, clusterStore } from "./cluster-store";
-import { extensionLoader } from "../extensions/extension-loader"
 import { appEventBus } from "./event-bus"
 import { ResourceApplier } from "../main/resource-applier";
+import { ipcMain } from "electron";
 
-export const clusterIpc = {
-  activate: createIpcChannel({
-    channel: "cluster:activate",
-    handle: (clusterId: ClusterId, force = false) => {
-      const cluster = clusterStore.getById(clusterId);
-      if (cluster) {
-        return cluster.activate(force);
-      }
-    },
-  }),
+if (ipcMain) {
+  handleRequest("cluster:activate", (event, clusterId: ClusterId, force = false) => {
```

Comment:
I'd like to avoid copy-pasting strings like `cluster:activate` over whole project..
Is it possible to wrap / encapsulate event-name like it was before?

Reply:
Agreed, let's see if I can refactor whole `cluster-ipc` module away.

Reply:
Strings are gone now.